frontend/src/app/application-header/application-header.component.spec.ts

```diff
@@ -0,0 +1,98 @@
+import { ComponentFixture, fakeAsync, TestBed, tick } from '@angular/core/testing';
```

Comment:
coverage of frontend/src/app/application-header/application-header.component.ts is 70%